cache/cache_bench_tool.cc

```diff
@@ -651,69 +675,78 @@ class CacheBench {
     uint64_t lookup_misses = 0;
     uint64_t lookup_hits = 0;
     // To hold handles for a non-trivial amount of time
-    Cache::Handle* handle = nullptr;
+    std::deque<Cache::Handle*> pinned;
+    size_t pin_count =
+        ((FLAGS_cache_size * FLAGS_pinned_ratio + FLAGS_value_bytes) /
+             FLAGS_value_bytes +
+         thread->tid) /
+        FLAGS_threads;
```

Comment:
Just for my own learning, why this formula is not just (FLAGS_cache_size * FLAGS_pinned_ratio) / FLAGS_threads?

Reply:
Modified slightly to make the calculation more clear. Thanks